README.md

```diff
@@ -279,6 +284,81 @@ We have a small collection of algorithms, one for every day of the month. Scroll
   - [More on the Caesar Cipher](http://practicalcryptography.com/ciphers/caesar-cipher)
 
 
+### **December 15 - Pascal's triangle**
+- **Problem**
```

Comment:
Include this line at the start of the problem:
"Pascal's Triangle is a triangle(represented as a 2-D array) in which the ends of the rows are 1. Each of the other values is equal to the sum of the two nearest numbers in the row above."